addon/components/tether-tooltip-on-element.js

```diff
@@ -62,24 +62,20 @@ export default TooltipAndPopoverComponent.extend({
         click to also trigger focusin */
 
         if (event !== 'click') {
-          $target.focusin(() => {
-            this.show();
-          });
+          $target.focusin(run.bind(this, this.show));
         }
 
-        $target.focusout(() => {
-          this.hide();
-        });
+        $target.focusout(run.bind(this, this.hide));
       }
 
-      $target.keydown((keyEvent) => {
+      $target.keydown(run.bind(this, (keyEvent) => {
```

Comment:
I didn't specifically have an issue here but all of these event handlers should really ensure their work is being done in a run loop.